lib/TickLib.sol

```diff
@@ -160,42 +160,41 @@ library LeafLib {
 
 }
 
-library TickLib {
+library LogPriceLib {
   // Could have an INVALID tick that is > 24 bits? But in `local` how do I write "empty"?
   int constant BP = 1.0001 * 1e18;
```

Comment:
does the change allow us to make BP smaller so we can support stablestable? The current change (as I understand it) allows higher prices to add support for more volatile unstableunstable. We discussed not putting more work into supporting stablestable, but what would the impact be if we decrease this number?

Reply:
Immediate impact is that total tick range must be increased to get the same price range as we have now, or we must introduce tick shift. Also future increased gas use when computing tickBase^logPrice (~a few hundred gas / tick I estimate)